.github/workflows/tests_primary.yml

```
@@ -207,7 +213,13 @@ jobs:
     - run: npm run build
     - name: Download Blob Reports from Azure Blob Storage
       run: |
-        az storage blob download-batch -d . -s '$web'  --pattern 'run-${{ github.run_id }}-${{ github.sha }}-test_test_runner/*.jsonl' --connection-string "${{ secrets.AZURE_CONNECTION_STRING_FOR_BLOB_REPORT }}"
+        run_dir='run-${{ github.run_id }}-${{ github.sha }}-test_test_runner'
```

```suggestion
        RUN_DIR='run-${{ github.run_id }}-${{ github.sha }}-test_test_runner'
```

lets have consistency here.